lib/ae/base/ae_template.py

```diff
@@ -10,17 +12,35 @@ def __init__(self, ufeSceneItem):
         self.attrS = ufe.Attributes.attributes(ufeSceneItem)
         super(AEBaseTemplate, self).__init__(ufeSceneItem)
 
-    def createSection(self, layoutName, attrList):
+    def createSection(self, layoutName, attrList, collapse=False):
         # We create the section named "layoutName" if at least one
         # of the attributes from the input list exists.
         for attr in attrList:
             if self.attrS.hasAttribute(attr):
-                with ufeAeTemplate.Layout(self, layoutName):
+                with ufeAeTemplate.Layout(self, layoutName, collapse):
                     # Note: addControls will silently skip any attributes
                     #       which does not exist.
                     self.addControls(attrList)
                 return
 
+    def createTransformAttributesSection(self):
+        # Get the xformOp order and add those attributes (in order)
+        # followed by the xformOp order attribute.
+        prim = mayaUsd.ufe.getPrimFromRawItem(self.item.getRawAddress())
+        geomX = UsdGeom.Xformable(prim)
+        xformOps = geomX.GetOrderedXformOps()
+        xformOpOrderNames = [op.GetOpName() for op in xformOps]
+        xformOpOrderNames.append(UsdGeom.Tokens.xformOpOrder)
```

Comment:
This creates a list with all the xform ops in order and then adds the xformOpOrder attribute at the end.